app/subscribers/proxy_config_event_subscriber.rb

```diff
@@ -3,7 +3,9 @@
 class ProxyConfigEventSubscriber
   def call(event)
     case event
-    when ProxyConfigs::AffectingObjectChangedEvent then ProxyConfigAffectingChangeWorker.perform_later(event.event_id)
+    when ProxyConfigs::AffectingObjectChangedEvent then
+      proxy = Proxy.find(event.proxy_id)
+      proxy.affecting_change_history.touch
```

Comment:
Should we maybe keep the exception catching and some of the error handling from the worker?
I think the chance that proxy would not be found here, or any other issue occur is minimized here - unlike in the case when it's done via a background job (where things can happen until the job gets executed, like the proxy can be deleted etc.). But not sure, maybe we're missing some scenario where an exception can indeed be thrown?

Reply:
I thought the same thing but, if an error happens here, won't it end up in logs and bugsnag anyway? I assumed there were no need to manually catch and report errors from here. In fact, I'm not sure it was needed in the previous code.